code/game/objects/items/stacks/medical.dm

```diff
@@ -91,7 +93,11 @@
 				if(!H.bleedsuppress) //so you can't stack bleed suppression
 					H.suppress_bloodloss(stop_bleeding)
 		if(affecting.status == BODYPART_ORGANIC) //Limb must be organic to be healed - RR
-			if(affecting.heal_damage(heal_brute, heal_burn))
+			var/actual_heal_brute = max(heal_brute - brute_heal_penalty * affecting.brute_dam,0)
+			var/actual_heal_burn = max(heal_burn - burn_heal_penalty * affecting.burn_dam,0)
+			if(actual_heal_brute + actual_heal_burn <= 0)

```

Comment:
```suggestion
			if(actual_heal_brute + actual_heal_burn == 0)
```

will never be less than 0

Reply:
might as well keep the top one to futureproof though?